src/libs/merge/symbols.map

```diff
@@ -1,5 +1,13 @@
 libelektra_0.8 {
         # kdbmerge.h
 	elektraMerge;
+	getNonOverlapOnlyBaseConflicts;
```

Comment:
Do you really want to have all this in the public API?

If yes, please reconsider names and let them start with elektra.

Reply:
> Do you really want to have all this in the public API?

I think so. I thought that if I use the functions to do calculations I could just use them for statistics later on. In the research question for my thesis we said that we want to take a look at different types of conflicts.

Reply:
The research question is not relevant for this decision, as we can always call these functions (even if they are private). The question here is if normal users that want to use the API need these functions? Isn't it enough from the user perspective to get an error/warnings with good text about why the merge failed?

Reply:
> as we can always call these functions (even if they are private).

I did not know that, thank you for the hint. As I agree to what you said here:

> Isn't it enough from the user perspective to get an error/warnings with good text about why the merge failed?

I'll make all those functions private :)

Reply:
It would be good if you make them private now as for public functions the review is much more strict.

Reply:
see doc/todo/API

Reply:
I just made all statistical functions static. Now only the main merge function is public.